assets/css/main.scss

```diff
@@ -265,3 +265,20 @@ div.localization
 .review-club-questions+ul details summary {
   font-style: italic;
 }
+
+// FIXME: combine all elements on the site using above
+// .review-club-questions class into users of the below .qa_details
+// class
+:not(.email).qa_details ul {
+  list-style-type: none;
+}
+
+.qa_details details {
+  border: thin solid silver;
+  margin: 2pt;
+  padding: 2pt;
+}
+
+.qa_details details summary {
```

Comment:
I think having a :hover style for the summary which changes the mouse to pointer would be a good usability addition.

Reply:
The hover style looks good!